docs/home/native-features.md

```diff
@@ -189,4 +189,113 @@ Here is an example of how to handle a shared URL:
+
+## Badging
+
+If your progressive web app is installed to the OS, the Badging API allows you to display a notification badge on your PWA's taskbar icon. You can use this functionality to inform the user when new content is available or requires their attention. This can help to keep the engagement up for your progressive web app - bringing users back to view new content.
+
+Here's what a badge for a progressive web app will look like on Windows:
+
+<div class="docs-image">
+   <img src="assets/home/native-features/badging-task-bar.png" alt="Image of a progressive web app with a notification badge displayed on Windows.">
+</div>
+
+### Displaying and Clearing Badges
+
+Using the Badging API only takes a few lines of code:
+
+```js
+if ('setAppBadge' in navigator) {
+  navigator.setAppBadge(1);
+}
+```
+
+In this code snippet, we are just checking if the `setAppBadge` feature is available and then making a call to `navigator.setAppBadge(1)` to display a badge with a value of one.
+
+To clear the app badge, use this function:
+
+```js
+navigator.clearAppBadge();
+```
+
+Or you can make another call to `setAppBadge()` with a value of zero:
+
+```js
+navigator.setAppBadge(0);
+```
+
+The Badging API can be used from withing your progressive web app, or it's service worker. A common use case for the Badging API is setting the badge in response to a `push` event, which can be handled with a listener in your service worker.
+
+Badging is often used in conjunction with the Notifications API to inform users when new content is available. The next section will take you through how make use of notifications for your progressive web app.
+
+## Push Notifications
+
+If you want a more direct way to notify users of content in your progressive web app, you can make use of the Notifications API. If the user gives permission for your app to send notifications, your app will be able to send a pop up notification that displays on the operating system regardless of whether or not your app is currently running.
+
+A notification displayed on Windows would look something like this:
+
+<div class="docs-image">
+   <img src="assets/home/native-features/notifications-action-center.png" alt="A push notification being sent by Edge for a progressive web app.">
+</div>
+
+### Displaying a Push Notification
+
+Notifications are often displayed in response to `push` event from the back end. A service worker can listen for this event and then display a notification when a push is received. 
+
+?> **Note** For simplicity sake, this guidance will focus on handling `push` from the front end, but you can learn more about the Push API [here.](https://microsoft.github.io/win-student-devs/#/30DaysOfPWA/advanced-capabilities/07?id=how-to-start-1)
+
+### Requesting Permission
```

Comment:
One question I had after reading this section is whether or not I have to ask for permission more than once?

Reply:
You shouldn't have to request permission twice, unless the user goes into their browser settings and revokes the permission for that URL.